pkg/frontend/nvme.go

```diff
@@ -116,6 +116,14 @@ func (s *Server) CreateNvmeSubsystem(_ context.Context, in *pb.CreateNvmeSubsyst
 		log.Printf("Already existing NvmeSubsystem with id %v", in.NvmeSubsystem.Name)
 		return subsys, nil
 	}
+	// check if another object exists with same NQN, it is not allowed
+	for _, item := range s.Nvme.Subsystems {
+		if in.NvmeSubsystem.Spec.Nqn == item.Spec.Nqn {
+			msg := fmt.Sprintf("Could not create NQN: %s since object %s with same NQN already exists", in.NvmeSubsystem.Spec.Nqn, item.Name)
+			log.Print(msg)
```

Comment:
Maybe `log.Printf("error: %v", msg)`? Usually we add "error:" prefix when we print errors